### PR TITLE
Fix/dependabot: ignore package update

### PR DESCRIPTION
**Pull request by Jorann**
I ignored all the providedCompile packages
Remove when PZ-592 is done

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -19,6 +19,16 @@ updates:
       timezone: "Europe/Amsterdam"
     reviewers:
       - "infonl/dimpact"
+    ignore:
+      # Ignore updates for these packages because it is needed for our current WildFly version (26.x)
```
Comment (bas-info-nl) [inline .github/dependabot.yml:23]: is this the only way to ignore these files? Seems convoluted...

**Review by bas-info-nl — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (Jorann) [reply .github/dependabot.yml:23]: It's temporary, we have wildfly update on the backlog PZ-592

Comment (edgarvonk) [reply .github/dependabot.yml:23]: No it's not temporary? We will always have these `providedCompile` dependencies. And they can never be upgraded by Dependabot. That's why we want to configure them here.

Comment (edgarvonk) [inline .github/dependabot.yml:23]: I would say:

"Ignore updates for these packages because they are provided by the WildFly application server."

Comment (edgarvonk) [reply .github/dependabot.yml:23]: Why is this convoluted you think? I think it's quite nice to do it this way. See: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore

Comment (Jorann) [reply .github/dependabot.yml:23]: :spongemock:

Comment (Jorann) [reply .github/dependabot.yml]: Changed

Comment (Jorann) [reply .github/dependabot.yml]: Okay it's not temporary no. But it might need to be changed when wildlfy is updated

Comment (edgarvonk) [reply .github/dependabot.yml]: Absolutely yes. Remove the 'might'. ;-) I expect that for every WildFly upgrade we will need to manually updated these deps.

**Review by edgarvonk — APPROVED** [review]

**Pull request merged**